syntax = "proto3";

package services;

message JsonRequest {
    string json_data = 1;
}

message JsonResponse {
    string json_data = 1;
    bool success = 2;
    string message = 3;
}

service JsonProcessor1 {
    rpc ProcessJson (JsonRequest) returns (JsonResponse);
}

service JsonProcessor2 {
    rpc ProcessJson (JsonRequest) returns (JsonResponse);
}
